README.md

```diff
@@ -127,6 +127,18 @@ Proposal failed verification. Please ensure the public key and merkle are correc
 **Note:** All politeia commands can dump the JSON output of every RPC command
 by adding the -json command line flag.
 
+Create a configuration file. For instance:
+```
+rpchost=127.0.0.1
+rpcuser=user
+rpcpass=pass
+rpccert=C:\Users\youruser\AppData\Local\Politeiad\https.cert
```

Comment:
I do not want to show windows paths.  Politeia[d-WWW] is and will not formally be supported on windows.

Isn;t the real issue here a lack of a proper and documented .conf file in politeaid and politeiawww?  I think we need to address that and then tell users where to find it instead of documenting a "wrong" example.

Reply:
@marcopeereboom is there an example in another repo for @rgeraldes to see how this is typically done?

Reply:
https://github.com/btcsuite/btcd/blob/master/sample-btcd.conf

We should do this both for d and www